### PR TITLE
Feature/nstack 184 terms

### PR DESCRIPTION
**Pull request by markusrubey**
- Add NStack Terms feature
- Add Terms to the demo project
- Point demo project to staging

### CODE REVIEW

File: demo/src/main/java/dk/nodes/nstack/demo/terms/TermsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+    }
+
+    private fun View.setVisibleOrGone(isVisible: Boolean) {
+        visibility = if (isVisible) {
+            View.VISIBLE
+        } else {
+            View.GONE
+        }
+    }
```
Comment (lucassales2) [inline demo/src/main/java/dk/nodes/nstack/demo/terms/TermsFragment.kt]: This works better as a var like in android ktx

var View.isVisible: Boolean 
get = visibility == View.VISIBLE
set(value) {
if(value) {
visibility = View.VISIBLE
} else {
visibility = View.GONE
}
}

File: demo/src/main/java/dk/nodes/nstack/demo/terms/TermsViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package dk.nodes.nstack.demo.terms
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import dk.nodes.nstack.kotlin.NStack
+
+class TermsViewModel : ViewModel() {
+
+    val viewState: MutableLiveData<TermsViewState> = MutableLiveData()
+
```
Comment (lucassales2) [inline demo/src/main/java/dk/nodes/nstack/demo/terms/TermsViewModel.kt]: Always facade the MutableLiveData to LiveData so the view can't set values on them

File: demo/src/main/java/dk/nodes/nstack/demo/terms/TermsFragment.kt (hunk 1 above)
Comment (markusrubey) [reply demo/src/main/java/dk/nodes/nstack/demo/terms/TermsFragment.kt]: Nice one, see 56126bd

File: demo/src/main/java/dk/nodes/nstack/demo/terms/TermsViewModel.kt (hunk 2 above)
Comment (markusrubey) [reply demo/src/main/java/dk/nodes/nstack/demo/terms/TermsViewModel.kt]: Fixed in b66df50

**Pull request merged**